ext/opcache/jit/zend_jit_ir.c

```diff
@@ -14591,6 +14597,13 @@ static int zend_jit_fetch_obj(zend_jit_ctx         *jit,
 			if (on_this) {
 				op1_info &= ~MAY_BE_RC1;
 			}
+			if (op1_info & (MAY_BE_RC1|MAY_BE_RCN)) {
+				if (!ssa_op) {
+					op1_info |= (MAY_BE_RC1|MAY_BE_RCN);
+				} else if (ssa_op->op1_def >= 0) {
+					op1_info |= (ssa->var_info[ssa_op->op1_def].type & (MAY_BE_RC1|MAY_BE_RCN));
+				}
+			}
```

Comment:
Instead of this I would set ``op1_info |= MAY_BE_RC1|MAY_BE_RCN`` in case we call "slow path".

```diff
diff --git a/ext/opcache/jit/zend_jit_ir.c b/ext/opcache/jit/zend_jit_ir.c
index 0082d531103..b300f31a7e4 100644
--- a/ext/opcache/jit/zend_jit_ir.c
+++ b/ext/opcache/jit/zend_jit_ir.c
@@ -14621,6 +14621,7 @@ static int zend_jit_fetch_obj(zend_jit_ctx         *jit,
 		ir_MERGE_list(slow_inputs);
 		jit_SET_EX_OPLINE(jit, opline);
 
+		op1_info |= MAY_BE_RC1 | MAY_BE_RCN; /* object may be captured/released in magic handler */
 		if (opline->opcode == ZEND_FETCH_OBJ_W) {
 			ir_CALL_1(IR_VOID, ir_CONST_FC_FUNC(zend_jit_fetch_obj_w_slow), obj_ref);
 			ir_END_list(end_inputs);
```